From f2573564bce89d93796014eb12311b78d00c8f35 Mon Sep 17 00:00:00 2001
From: Thomas Gleixner <tglx@linutronix.de>
Date: Fri, 24 Jul 2009 18:07:22 +0200
Subject: [PATCH] mm: vmscan: do not disable interrupts on -RT

commit 104f75cb1a751a023beddacf56ca6c19ed90ce6c in tip.

[PG: 33rt retains just this one line of the original tip change]

Signed-off-by: Thomas Gleixner <tglx@linutronix.de>
Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
---
 mm/vmscan.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/mm/vmscan.c b/mm/vmscan.c
index 5c46206..48700b6 100644
--- a/mm/vmscan.c
+++ b/mm/vmscan.c
@@ -1269,7 +1269,7 @@ static unsigned long shrink_inactive_list(unsigned long max_scan,
 
 		nr_reclaimed += nr_freed;
 
-		local_irq_disable();
+		local_irq_disable_nort();
 		if (current_is_kswapd())
 			__count_vm_events(KSWAPD_STEAL, nr_freed);
 		__count_zone_vm_events(PGSTEAL, zone, nr_freed);
-- 
1.7.0.4

